metakb/transform/civic.py

```diff
@@ -616,44 +675,32 @@ def _get_therapy_descriptors(self, drug):
         """
         label = drug['name']
         ncit_id = f"ncit:{drug['ncit_id']}"
-        therapy_norm_resp = None
+        highest_match = 0
+        normalized_therapy_id = None
 
         for query in [ncit_id, label]:
             if not query:
                 continue
 
             therapy_norm_resp = self.therapy_query_handler.search_groups(query)
-            if therapy_norm_resp['match_type'] != 0:
-                break
+            if therapy_norm_resp['match_type'] > highest_match:
+                highest_match = therapy_norm_resp['match_type']
+                normalized_therapy_id = therapy_norm_resp['value_object_descriptor']['value']['therapy_id']  # noqa: E501
```

Comment:
not sure if you plan to do it in other branch but `therapy_id` should be changed to `id` as the normalizer was updated